x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/syncjob/action/RestListConnectorSyncJobsAction.java

```diff
@@ -40,11 +42,16 @@ protected RestChannelConsumer prepareRequest(RestRequest restRequest, NodeClient
         String connectorId = restRequest.param(ListConnectorSyncJobsAction.Request.CONNECTOR_ID_FIELD.getPreferredName());
         String statusString = restRequest.param(ConnectorSyncJob.STATUS_FIELD.getPreferredName());
         ConnectorSyncStatus status = statusString != null ? ConnectorSyncStatus.fromString(statusString) : null;
+        String jobTypeString = restRequest.param(ConnectorSyncJob.JOB_TYPE_FIELD.getPreferredName());
```

Comment:
Could we use `paramAsStringArray` here? You can even pass `null` as default value there.

Reference: https://github.com/elastic/elasticsearch/blob/7764fdb3ea93ed329cf26d20000fa7cef1e7c0bb/server/src/main/java/org/elasticsearch/rest/RestRequest.java#L473

Reply:
Good idea, updated.